### ISSUE TITLE
Unable to update number attribute

### ISSUE THREAD

**Issue by aneilbaboo** [ryanfitz/vogels#206]
I'm using Vogels 2.2.0.  

The problem is that I'm unable to get a joi.number attribute to update.

The key code is shown below
```javascript
var TestModel = vogels.define('TestModel', {
  hashKey : 'id',
  schema : {
    id      : joi.string(),
    num     : joi.number()
  }
});

var test = new TestModel({id: "1"});

test.save(function (err, tSaved) {
      console.log("Attempting to update num to 100");
      tSaved.update({num: 100}, function (err, tUpdated) {
        if (tUpdated.get('num')==100) {
          console.log("Successfully updated test.num => 100!");
        } else {
          console.log("Failed to update test instance: %j", tUpdated.attrs);
          // => this is what gets printed
        }

      });
    });

```

I created a repo which allows you to test the problem, here: https://github.com/aneilbaboo/vogelsupdateproblem/

**Comment by iliasbhal** [comment]
how did you solve this issue? I have the same problem.
thanks ;)

**Comment by aneilbaboo** [comment]
Simple fix.  I was misusing the API.

`.update` is a Class method which is invoked by providing the primary key in the list of attributes.  When you already have an instance, simply use `.save`.  I was trying to do an $add operation, with an existing instance.  I'm not sure whether `.save` works with $add; instead, I switched from:

```javascript
test.update({ myfield: { $num: 1}}); // wrong
```
to
```javascript
Test.update({ test.get('id'), num: {$ add: 1}}); // id is primary key 
```

By the way, I just noticed that this library is no longer maintained (#171), and that activity has moved to a fork called Dynogels: https://github.com/clarkie/dynogels.
